src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/repository/PCCRegionRepository.kt

```diff
@@ -0,0 +1,8 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.repository
+
+import org.springframework.data.repository.CrudRepository
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.PCCRegion
+
+interface PCCRegionRepository : CrudRepository<PCCRegion, String> {
```

Comment:
nitpick: this should be `CrudRepository<PCCRegion, PCCRegionID>`

Reply:
you are right - I thought I'd got them all - will update